setup/scaling/caching/configure-redis-usage.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: single
+title:  "Configure Redis Usage"
```

Comment:
Instead of "Usage" maybe something more specific, like "Configure Redis to scale up Spinnaker" or something more exact about what you're configuring Redis for?

Reply:
See my comment below -- it's not that clear cut. We could say "Configure Spinnaker's usage of Redis" maybe?